src/lib/Security/AuthorizationHeaderRESTRequestMatcher.php

```diff
@@ -6,15 +6,21 @@
  */
 declare(strict_types=1);
 
-namespace Ibexa\Contracts\Rest\Security;
+namespace Ibexa\Rest\Security;
 
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\RequestMatcher;
 
+/**
+ * @internal
+ */
 final class AuthorizationHeaderRESTRequestMatcher extends RequestMatcher
```

Comment:
So, as a conclusion to the discussion both here and internal, we'd just want here to mention that this is used by semantic (firewall?) configuration in custom projects.

Reply:
Resolved via https://github.com/ibexa/rest/pull/101/commits/6a26ba55809b2eb345ee96e36c5b955787ff8bad.